hedera-mirror-rest/utils.js

```
@@ -832,13 +836,13 @@ const getTransactionTypeQuery = async (parsedQueryParams) => {
     return '';
   }
 
-  const protoId = await transactionTypes.get(transactionType);
-  if (protoId !== undefined) {
+  try {
+    const protoId = await transactionTypes.get(transactionType);
     return `${constants.transactionColumns.TYPE}${opsMap.eq}${protoId}`;
+  } catch (err) {
+    // throw error if transactionType filter was provided but invalid
+    throw new InvalidArgumentError(`Invalid transactionType value '${transactionType}'`);
```

Would prefer to directly throw `InvalidArgumentError` from `transactionTypes.get()`

---

The only caveat to that is `transactionTypes.get()` previously was/could someday also be used internally for building queries, so in theory the error could be caused by bad code on our part and not actually user error (400 error vs 500).  Not that we should be releasing code with such a major bug.  I'll make the change as for now it's not an issue, just something to keep in mind.

---

Done.

---

This catch is still here

---

Ah, whoops, I added a catch in two locations and removed the wrong one.  This one has been removed now, `isValidTransactionType()` catch has been put back as it does its own error logic later on.